types/chai/index.d.ts

```diff
@@ -1653,10 +1615,10 @@ declare namespace Chai {
 
 declare const chai: Chai.ChaiStatic;
 
-declare module "chai" {
-    export = chai;
-}
+export = chai;
```

Comment:
It seems that removing "chai" module or namespace breaks most downstream typings for chai matchers (like `chai-as-promised` and `chai-string`).

I had to pin to v4.0.6 to avoid this breakage.

Reply:
Oh dear. How did this pass the Travis build?

Reply:
Dunno--what's your code look like to have this diff work for you? I've always done something like

```ts
import { expect } from "chai"
```

(which doesn't work with v4.0.7, even with `allowSyntheticDefaultImports: true`).

Reply:
@segayuu ^^

Reply:
I'm sorry to destroy the export structure.
I was thinking that it was okay because I had successful local tests in advance in advance.
dtslint intends that it is desirable to invalidate export of interface, type which is probably meaningless except for type definition.